Formula/fio.rb

```diff
@@ -16,6 +16,13 @@ class Fio < Formula
 
   uses_from_macos "zlib"
 
+  # Upstream fix for "ld: weak import of symbol '___darwin_check_fd_set_overflow' not supported because of option: -no_weak_imports for architecture x86_64"
```

Comment:
Lines should be `118` characters at most. You can always continue the comment on the next line

Reply:
I've just pushed an update for that very thing :-)